backend/src/auth/interfaces/signup-user.interface.ts

```diff
@@ -2,5 +2,11 @@ import { User } from '@prisma/client';
 
 export type SignupUser = Omit<
```

Comment:
これ、新規作成する時の型なので、
```suggestion
export type SignupUser = Omit<id, createAt, updatedAt, twoFactorAuthSecret>
```
でいいと思います！
auth.controller.tsの42ログインするとこ、nickname = name, onlineStatus=true, isTwoFactorAuthEnabled = false で修正お願いします。せきぐちさんと関係ないとこもありますが。

Reply:
修正しました。